concreate/descriptor.py

```diff
@@ -105,10 +111,24 @@ def _process_labels(self):
         """ Generate labels from concreate keys """
         if "labels" not in self.descriptor:
             self.descriptor['labels'] = []
-        if "description" in self.descriptor:
-            self.descriptor['labels'].append({'name': 'description',
-                                              'value': self.descriptor['description']})
 
+        # The description and maintainer keys available in image descriptor
+        # root are added as labels to the image
+        for key in ['maintainer', 'description']:
+            # If we define the maintainer or description label in the image descriptor
```

Comment:
we should decide where maintainer is specified in our tool - i would remove maintainer instruction from concreate and use only labels to be consistent.

Reply:
The `maintainer` label is specific to Docker container image and it's guidelines. I think it is generic enough to leave it as a first class key.

Reply:
Its deprecated in Dockerfile, and image spec from OCI doesn't have it either - i don't think we should implement special key for it. We can generate Dockerfile key but we should have one way of specifying it.

Reply:
I can remove the maintainer key.

Reply:
Created: https://github.com/jboss-container-images/concreate/issues/23

Reply:
ok, thanks, approving PR. so merge it when you feel it should be merged

Reply:
OCI spec defines "author" as top level key, not label: https://github.com/opencontainers/image-spec/blob/master/config.md#properties